airflow-core/src/airflow/serialization/serializers/pydantic.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from importlib import import_module
+from typing import TYPE_CHECKING, cast
+
+from pydantic import BaseModel
+
+from airflow.serialization.serde import _is_pydantic_basemodel
+from airflow.utils.module_loading import qualname
+
+if TYPE_CHECKING:
+    from airflow.serialization.serde import U
+
+serializers = [
+    "pydantic.main.BaseModel",
+]
+deserializers = serializers
+stringifiers = serializers
+
+__version__ = 1
+
+
+def _resolve_pydantic_class(qn: str):
```

Comment:
I missed this one, but let's not re-invent the wheel and use "import_string" from airflow.utils.module_loading